slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/SignedBlocksDao.java

```diff
@@ -96,4 +96,12 @@ public void insertBlockProposal(final Handle handle, final SignedBlock signedBlo
             "SELECT validator_id, slot, signing_root FROM signed_blocks WHERE validator_id = ?")
         .bind(0, validatorId).mapToBean(SignedBlock.class).stream();
   }
+
+  public Optional<UInt64> maximumSlot(final Handle handle, final int validatorId) {
```

Comment:
The minimumSlot function shouldn't be needed anymore now can be removed

Reply:
done